packages/builder/src/registry.ts

```diff
@@ -152,6 +152,8 @@ export class FallbackRegistry extends EventEmitter implements CannonRegistry {
               `you can verify this if you have access to the node logs. \n\n ${err} \n ${err.error}`
           );
         }
+
+        throw new Error(err);
```

Comment:
Not sure on context, but wold be better to do `throw err` instead? to not lose the original stacktrace.

Reply:
I think the error messaging here is fine. We include the original error data, although may be best to use `toString`.